SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/Connection/SpatialConnectionManagerTest.cpp

```diff
@@ -365,3 +365,19 @@ CONNECTIONMANAGER_TEST(SetupFromCommandLine_Receptionist_URLAndExternalBridge)
 
 	return true;
 }
+
+CONNECTIONMANAGER_TEST(SetupFromCommandLine_Receptionist_SetupConnectionConfigFromURL)
+{
+	// GIVEN
+	FTemporaryCommandLine TemporaryCommandLine("\\");
+	USpatialConnectionManager* Manager = NewObject<USpatialConnectionManager>();
+
+	// WHEN
+	const bool bSuccess = Manager->TrySetupConnectionConfigFromCommandLine("SomeWorkerType");
+
+	// THEN
+	TestEqual("Success", bSuccess, false);
+	TestEqual("WorkerType", Manager->ReceptionistConfig.WorkerType, "SomeWorkerType");
```

Comment:
Perhaps we should not actually set the worker type on the receptionist config. All other paths (locator, dev auth) do not set the worker type if they fail to load from command line. 

We could remove this check and alter the logic inside USpatialConnectionManager::TrySetupConnectionConfigFromCommandLine so that is only sets the worker type if ReceptionistConfig.TryLoadCommandLineArgs() succeeds.

Reply:
Done!

Reply:
Saw your local changes but it doesn't seem they got pushed?

Reply:
oh hm I got this now:

`Pushing to https://github.com/spatialos/UnrealGDK.git
To https://github.com/spatialos/UnrealGDK.git
 ! [rejected]          Bugfix_UNR4117 -> Bugfix_UNR4117 (fetch first)
error: failed to push some refs to 'https://github.com/spatialos/UnrealGDK.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.`

Reply:
I'll pull and retry

Reply:
Yeah, interesting. You are the only one writing to this branch so ???????? Let me know if there is an issue with that.

Reply:
I think its cause I didn't push before committing the release note

Reply:
Or this *bennetty* issue